ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/CreateEntityTaskTest.groovy

```diff
@@ -61,7 +61,7 @@ class CreateEntityTaskTest extends BaseTest {
         result.task(":hubDeployUserArtifacts").outcome == SUCCESS
 
         File entityFile = Paths.get(testProjectDir.root.toString(), "entities", "my-new-entity.entity.json").toFile()
-        entityFile.isFile() == true
+        entityFile.isFile()
```

Comment:
What is this line of code doing - should it just be removed?

Reply:
The Spock testing framework does not require a condition check for boolean assertions.